eis_toolkit/transformations/one_hot_encoding.py

```diff
@@ -0,0 +1,104 @@
+from numbers import Number
+from typing import Literal, Optional, Sequence, Union
+
+import numpy as np
+import pandas as pd
+from sklearn.preprocessing import OneHotEncoder
+
+from eis_toolkit import exceptions
+from eis_toolkit.utilities.checks.dataframe import check_columns_valid
+
+
+def one_hot_encode(
+    data: Union[pd.DataFrame, np.ndarray],
+    columns: Optional[Sequence[str]] = None,
+    drop_encoded_columns=True,
```

Comment:
I would name this parameter `drop_original_columns` since they are dropped, not the encoded.